requirements.txt

```diff
@@ -0,0 +1,4 @@
+argcomplete==1.9.5
```

Comment:
i am not sure if all buildtest deps are captured in this requirements.txt file. If it makes sense to have a ``docs/requirements.txt`` for building docs and regression test then that could be fine with me. Originally i have everything in one so i can do the testing, doc-build.

Reply:
I think it's okay having one too, and it's more appropriate in the docs directory since most of the requirements are for that. The setup.py / pip install -e . would handle installing the requirements via the setup.py using pip.